REFERENCE.md

```diff
@@ -0,0 +1,359 @@
+            init = new Init("questions", securityMap, consumerSecret, req);
+
+            // Call the generate() method to retrieve a JavaScript object
+            String questionJson = init.generate();
+
+            // Pass the object to the initialisation of any Learnosity API
+            // For instance in your jsp file you can have:
+            // <script src="//questions.learnosity.com"></script>
+            // var questionsApp = LearnosityApp.init(questionJson);
+
+            System.out.println(questionJson);
+        }catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+}
+```
+
+Raw JSON initialisation snippet:
+
+```java
+import learnositysdk.request.Init;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class Questions {
+    public static void main (String[] args) throws Exception
+    {
+        try {
+            Init init;
+            String consumerSecret = "74c5fd430cf1242a527f6223aebd42d30464be22";
+
+            String secString = "{\"consumer_key\":\"yis0TYCu7U9V4o7M\","
+                    +    "\"domain\": \"assess.vg.learnosity.com\","
+                    +    "\"user_id\": \"$ANONYMIZED_USER_ID\"}";
+
+            String reqString = "{\"state\":\"initial\","
+                    +  "\"type\":\"local_practice\","
+                    +  "\"response_id\":\"60005\","
+                    +  "\"questions\":"
+                    +   "[{\"stimulus_list\":"
+                    +     "[\"London\","
+                    +        "\"Dublin\","
+                    +        "\"Paris\","
+                    +      "\"Sydney\"],"
+                    + "\"stimulus\":\"Match the cities to the parent nation\","
+                    +   "\"type\":\"association\","
+                    +   "\"possible_responses\":"
+                    +   "[\"Australia\","
+                    +     "\"France\","
+                    +     "\"Ireland\","
+                    +       "\"England\"],"
+                    +   "\"validation\":"
+                    +   "{\"valid_responses\":"
+                    +       "[\"England\","
+                    +             "\"Ireland\","
+                    +       "\"France\","
+                    +       "\"Australia\"]}}]}";
+            init = new Init("questions", secString, consumerSecret, new JSONObject(reqString));
+            String questionJson = init.generate();
+            System.out.println(questionJson);
+
+            // Now you can pass questionJson to the initialisation of Questions API
+            // For instance on jsp you might have
+            // <script src="//questions.learnosity.com"></script>
+            // var questionsApp = LearnosityApp.init(questionJson);
+        }catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+}
+```
+
+#### Arguments
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
#### Arguments

```